R/db.R

```diff
@@ -189,45 +343,84 @@ write_data_safely <- function(dbconn, tblname, df, key) {
     }}
   }
 
-  if (!DBI::dbExistsTable(dbconn, tblname))
-    return(write_column_hashed_data(df, append = FALSE))
+  ## Use transactions!
```

Comment:
Not sure if I got the syntax correctly, and whether we want to start the transaction earlier, starting in `cmiyc::execute()`